Tasks/SonarQubePostTest/PRCA/ReportProcessor-Module.psm1

```diff
@@ -28,12 +28,19 @@ function FetchAnnotatedNewIssues
     CreateComponentKeyToPathMap $json
 
     # '@' makes sure the result set is returned as an array
-    $newIssues = @($json.issues | Where {$_.isNew -eq $true})
+    $newIssues = @($json.issues | Where { $_.isNew -eq $true })
     Write-Host "SonarQube found $($json.issues.Count) issues out of which $($newIssues.Count) are new"
+    
+    $newFileLevelIssues = @($newIssues | Where {(IsFileLevelIssue $_)})
```

Comment:
This is the actual bug fix - ignore the assembly level issues. In the future we may want to post the issues somewhere else, but for now we decided to ignore them. Note that only FxCop reports assembly level issues for C#.